themes/digital.gov/src/scss/new/shortcodes/_note.scss

```diff
@@ -1,139 +1,131 @@
 @use "uswds-core" as *;
 
-.note {
-  @include u-measure(6);
-  @include u-margin-bottom(2);
-  @include u-padding-y(2.5);
-  @include u-padding-x(3);
-  @include u-text("light");
-  @include u-radius("sm");
-  @include u-bg("blue-5");
-  @include u-text("blue-70");
-  @include u-border-left("05", "solid", "blue-20");
-
-  h4 {
-    @include u-margin-bottom("105");
-    @include u-margin-top(0);
-    @include u-font("sans", "lg");
-    @include u-text("light");
-
-    i {
-      @include u-margin-right("05");
-    }
-  }
+.dg-note {
+  background-color: color("blue-5");
+  border-left-color: color("blue-20");
+  border-left-style: solid;
+  border-left-width: 5px;
+  color: color("blue-70");
+  font-size: size("ui", "md");
+  font-weight: font-weight("light");
+  padding: units(3);
 
   a {
```

Comment:
Replaced `dg-note__header` with the `h4` element to override the `usa-prose` styles. Removed `!important`!

Reply:
@nick-mon1 @clmedders I thought we were going to revert this change?